docs/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-historie-bevragen API als release is uitgebracht zijn de specificaties van deze endpoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar beschikbaar geweest via de repository. In de [v0.9.0 release van BRP-bevragen](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v0.9.0) zijn de historie-endpoints nog opgenomen. In de afgelopen maanden hebben we de historie-endpoints in een aparte API ondergebracht. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Recentelijk hebben we de specificaties ook aangepast op het hergebruik van componenten uit BRP-Bevragen release 1.2.0.
+Dit is een globaal (en niet limitatief) overzicht van wijzigingen die daarvoor zijn doorgevoerd.
+
+### Openapi.yaml :
+
+- server-url is aangepast (omdat het een aparte API is geworden.) --> https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen _**(breaking)**_
+- OperationId's zijn aangepast ivm code-generatie issues en nu in UpperCamelCase. _**(breaking)**_
```

Comment:
@MelvLee moeten we dit breaking noemen? Een API die op de vorige versie is gebouwd zal hier niet op breken, tot deze zelf opnieuw code genereert. Of zie ik dat verkeerd?

Reply:
@fsamwel, wat definieren we als breaking? Functioneel is het niet breaking, want oude consumers kunnen met de nieuwe provider werken. Maar als ik als consumer of provider met de nieuwe openapi.yaml opnieuw de plumbing code genereer, dan krijgen ze een compile error omdat de oude methode is hernoemd.

Reply:
Ik ben geen Melvin, maar volgens mij klopt jouw conclusie. Ik haal de toevoeging -**breaking**_ hier alvast weg. Mocht Melvin het daar niet mee eens zijn, dan hoor ik dat graag.

Reply:
@CathyDingemanse zullen we dit maandag even bespreken?

Reply:
Het is niet dat ik er niet mee eens ben. Het is meer wat je als breaking definieer. Je geeft bijv. aan dat het veranderen van de server url breaking is. Ik vind van niet. Ik hoef namelijk als consumer geen code aanpassing te doen, slechts configuratie. Als ik het goed gedaan heb, dan hoef ik de url alleen op de API Gateway aan te passen en alle consumers voor de gateway hoeven niet te weten dat de server url is gewijzigd

Reply:
breaking is wanneer een werkende client door de gewijzigde API (specificaties/interface) niet meer (goed) werkt.